fiftyone/utils/lerobot.py

```
@@ -0,0 +1,1549 @@
+        ) from exc
+
+    return _build_resolved_media_asset(
+        reference,
+        asset,
+        path=path,
+        size_bytes=size_bytes,
+        media_type=media_type,
+    )
+
+
+def _open_parquet(path, role):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_open_parquet`

(ANN202)